build/BuildOutputs.props

```diff
@@ -3,5 +3,7 @@
     <DotnetHostBaseName>dotnet$(ExeExtension)</DotnetHostBaseName>
     <DotnetHostFxrBaseName>$(DynamicLibPrefix)hostfxr$(DynamicLibExtension)</DotnetHostFxrBaseName>
     <HostPolicyBaseName>$(DynamicLibPrefix)hostpolicy$(DynamicLibExtension)</HostPolicyBaseName>
+    <ArchiverDll>$(ToolsOutputDirectory)/Archiver.dll</ArchiverDll>
```

Comment:
This is not really a Build Output. It's an intermediate file that we use to produce the real build output. I'd keep this in the LZMA csproj as a static [outside of target] property.